### ISSUE TITLE
 MM decomp licensing #1 

### ISSUE THREAD

**Issue by louist103** [louist103/mm-licensing#1]
Hi!

You are mentioned here because you previously contributed to the [zeldaret/mm repo](https://github.com/zeldaret/mm), a decompilation of the game Legend of Zelda Majoras Mask.

There is a move to license the source of that repo under [CC0](https://creativecommons.org/publicdomain/zero/1.0/), and for that we need your permission, as past contributors.

For more information: https://github.com/louist103/mm-licensing/blob/main/README.md

If you don't care, the best choice to do for the project and users of the project (researchers, fans, modders...) is to agree.
If you agree, you can reply to this issue with some message like "I agree to share all my contributions to the mm decomp under CC0" (please include OOT in the statement if you also helped with OOT decomp)
If you disagree, please get in touch! Through [github issues](https://github.com/louist103/mm-licensing/issues) or [discord](https://discord.gg/rDvMsgnU2m)

**Comment by louist103** [comment]
@louist103 @Kelebek1 @Rozelette @hensldm @engineer124 @EllipticEllipsis @AngheloAlf @tom-overton @sonicdcer @kyleburnette @Kenix3 @jpburnett  @isghj5 @Thar0 @mzxrules @zbanks @ordlucas @shawlucas @rylieb @petrie911 @chloebangbang @LtPeriwinkle @retro-git @Zelllll @mmbx304 @fig02 @MrPolymorph @thaumavorio @Cleverking2003 @ethteck @TrulyNaN @pocable @Andzura @immibis @emilybrooks @NickSturgeon @Synray @Roman971 @StickyThwomp @CrookedPoe @Averane1 @MMDecomp @krm01 @krj9c @SailorSnoW @waffleoRai @Dragorn421 @nckwntzl @quaantuum @ZoeyZolotova @sjossi @Electrk @gamestabled @WonderJ13 @dhustkoder

**Comment by louist103** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by nckwntzl** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by louist103** [comment]
> @sjossi @Electrk @gamestabled @WonderJ13 @dhustkoder

@sjossi @Electrk @gamestabled @WonderJ13 @dhustkoder

**Comment by Elletra** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by Elletra** [comment]
Might wanna replace "oot" with "mm"

**Comment by tom-overton** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by ethteck** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by WonderJ13** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by LtPeriwinkle** [comment]
I agree to share all of my contributions to the MM decomp under CC0.

**Comment by MaxBartlett** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by TrulyNaN** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by isghj5** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by Dragorn421** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by AngheloAlf** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by StickyThwomp** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0.

**Comment by sjossi** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0.

**Comment by ordlucas** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by krm01** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by gamestabled** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by sonicdcer** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by waffleoRai** [comment]
I agree to share all my contributions to the oot and mm decomps under CC0

**Comment by fig02** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by Thar0** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by retro-meister** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by pocable** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by SailorSnoW** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by zbanks** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by Andzura** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by Averane1** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by hensldm** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by Synray** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by Kelebek1** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by engineer124** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by immibis** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by immibis** [comment]
[Why CC0?](https://github.com/louist103/mm-licensing/issues/2)

**Comment by immibis** [comment]
~~I agree to share all my contributions to the oot decomp and mm decomp under CC0.~~

~~But I don't think it's the greatest idea. I only agree because there aren't many and they don't really matter.~~

[see below](https://github.com/louist103/mm-licensing/issues/1#issuecomment-1396697095)

**Comment by MrPolymorph** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under
CC0.

**Comment by krimtonz** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under
CC0.

**Comment by Rozelette** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under
CC0.

**Comment by immibis** [comment]
> ~I agree to share all my contributions to the oot decomp and mm decomp under CC0.~
> 
> ~But I don't think it's the greatest idea. I only agree because there aren't many and they don't really matter.~

Retracted, because the moderators retaliated against me for questioning the policy.

**Comment by Kenix3** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by louist103** [comment]
@swinginman Hey! you have a PR that seems close to getting merged. Please see the above messages.

**Comment by thaumavorio** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by krj9cr** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by shawlucas** [comment]
I agree to share all my contributions to the oot and mm decomps under CC0

**Comment by louist103** [comment]
@francisjeanneau You also have a PR that is going in soon. Please see this thread.

**Comment by francisjeanneau** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by ZoeyZolotova** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by swinginman** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by dhustkoder** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0

**Comment by Zakru** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by KidFeroz** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by nussico** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by SeDan01** [comment]
I agree to share all my contributions to the mm decomp under CC0"

**Comment by HailToDodongo** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by AbdooOwd** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by MegaIDK** [comment]
i agree to share all my contributions to the mm decomp under CC0

**Comment by immibis** [comment]
I will sooner or later be restricted from commenting, then delete this account due to Github's new 2FA rules. Since nobody has reached out to apologize and seek permission in the last several months, I suppose you won't be able to ask me for it any more.

**Comment by comportase** [comment]
Have @jpburnett, @EllipticEllipsis, @Roman971, @mzxrules, @CrookedPoe, @Zelllll and others agreed to license yet? Harkinian are being held back by this issue.

**Comment by ethteck** [comment]
@comportase As I said [here](https://github.com/Dragorn421/oot-licensing/issues/1#issuecomment-1707647373), if you have questions, please reach out to the project organizers instead of publicly trying to pressure people here.

**Comment by zrnorth** [comment]
I agree to share all my contributions to the mm decomp under CC0.

**Comment by Wimmerrad** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by codeOverFlow** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by Javarooster-png** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by garrettjoecox** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by swinginman** [comment]
> @swinginman Hey! you have a PR that seems close to getting merged. Please see the above messages.

Sorry I'm so late!

I agree to share all my contributions to the mm decomp under CC0

Is there anything else I need to do?

**Comment by GinNoOokami** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by Archez** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by inspectredc** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by t0mtee** [comment]
I agree to share all my contributions to the mm decomp under CC0!

**Comment by Eblo** [comment]
I agree to share all my contributions to the mm decomp under CC0

**Comment by JordanLongstaff** [comment]
I agree to share all my contributions to the oot and mm decomp projects under CC0.

**Comment by Ozero4** [comment]
I agree to share all my contributions to the oot and mm decomp projects under CC0.

**Comment by mzxrules** [comment]
I agree to share my contributions to the mm decomp under CC0.
Additionally, I agree to release past contributions (prior to March 25th, 2026) to the mm decomp under CC0.

**Comment by djevangelia** [comment]
I agree to share all my contributions to the MM decomp under CC0 (including past and future contributions to the OoT decomp previously agreed to)

**Comment by AACooper1** [comment]
I agree to share all my contributions to the MM and OoT decomp projects under CC0.

**Comment by Zeri-Ai** [comment]
I agree to share all my contributions to the oot decomp and mm decomp under CC0.
Incidentally, the More information link throws a 404 to me.

**Comment by Dragorn421** [comment]
> Incidentally, the More information link throws a 404 to me.

fixed! ty
